packages/core/admin/admin/src/pages/MarketplacePage/tests/plugins.test.js

```diff
@@ -140,12 +146,254 @@ describe('Plugins tab', () => {
   });
 
   it('shows plugins filters popover', () => {
-    render(App);
+    const filtersButton = screen.getByRole('button', { name: /filters/i });
 
+    userEvent.click(filtersButton);
+    const collectionsButton = screen.getByRole('button', { name: 'No collections selected' });
+    const categoriesButton = screen.getByRole('button', { name: 'No categories selected' });
+
+    expect(collectionsButton).toBeVisible();
+    expect(categoriesButton).toBeVisible();
+  });
+
+  it('shows the collections filter options', () => {
     const filtersButton = screen.getByRole('button', { name: /filters/i });
-    fireEvent.click(filtersButton);
-    screen.getByLabelText(/no collections selected/i);
-    screen.getByLabelText(/no categories selected/i);
-    fireEvent.click(filtersButton);
+    userEvent.click(filtersButton);
+
+    const collectionsButton = screen.getByRole('button', { name: 'No collections selected' });
+
+    userEvent.click(collectionsButton);
+
+    const mockedServerCollections = {
+      'Made by official partners': 9,
+      'Made by Strapi': 13,
+      'Made by the community': 69,
+      Verified: 29,
+    };
+
+    Object.entries(mockedServerCollections).forEach(([collectionName, count]) => {
+      const option = screen.getByRole('option', { name: `${collectionName} (${count})` });
+      expect(option).toBeVisible();
+    });
+  });
+
+  it('shows the categories filter options', () => {
+    const filtersButton = screen.getByRole('button', { name: /filters/i });
+    userEvent.click(filtersButton);
+
+    const categoriesButton = screen.getByRole('button', { name: 'No categories selected' });
+
+    userEvent.click(categoriesButton);
+
+    const mockedServerCategories = {
+      'Custom fields': 4,
+      Deployment: 2,
+      Monitoring: 1,
+    };
+
+    Object.entries(mockedServerCategories).forEach(([categoryName, count]) => {
+      const option = screen.getByRole('option', { name: `${categoryName} (${count})` });
+      expect(option).toBeVisible();
+    });
+  });
+
+  describe('filter requests', () => {
```

Comment:
ℹ️  I added a new describe to so other tests can still benefit from the cache

Reply:
great idea